aptos-move/aptos-vm/src/keyless_validation.rs

```diff
@@ -192,7 +218,32 @@ pub(crate) fn validate_authenticators(
     };
 
     for (pk, sig) in authenticators {
-        let jwk = get_jwk_for_authenticator(&patched_jwks, pk, sig)?;
+        let jwk = match pk {
+            // 1. This is a federated keyless account; look in 0x1 and then in `jwk_addr` for JWKs
+            AnyKeylessPublicKey::Federated(fed_pk) => {
+                // 1.a: Try finding the JWK in 0x1.
+                match get_jwk_for_authenticator(&patched_jwks, pk.inner_keyless_pk(), sig) {
```

Comment:
can we combine the 

`get_jwk_for_authenticator(&patched_jwks, pk.inner_keyless_pk(), sig)`

and 

`AnyKeylessPublicKey::Normal(pk) => get_jwk_for_authenticator(&patched_jwks, pk, sig)?` 

call below in line 240 into one?  and then in the case of err do the federated lookup if it is a federated key?

Reply:
Eh maybe. I kinda wish you did it so we can see if it looked any better.